models/tests/test_refugee.py

```diff
@@ -40,64 +38,54 @@ def test_invalid_numOfFamilyMember1(self):
         with self.assertRaises(Refugee.InvalidNumOfFamilyMemberException):
             Refugee(firstname="Tom",
                     lastname="Bond",
-                    camp=Camp("UCL"),
                     num_of_family_member=-10,
-                    starting_date= date(2020,1,2),
-                    medical_condition_type=Refugee.MedicalCondition.HIV)
+                    starting_date=date(2020, 1, 2),
+                    medical_condition_type=[Refugee.MedicalCondition.HIV])
 
     def test_invalid_firstname(self):
```

Comment:
Can we add: 

"""
Test invalid first name. 
"""

for consistency please? 

And similarly for the tests below.